package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ssv-scanner",
-  "version": "0.0.4",
-  "description": "Library to retrieve cluster snapshots and owner nonce from the ssv.network contract.",
+  "version": "1.0.0",
+  "description": "Tool for retrieving events data (cluster snapshots and registration nonce) from the SSV network contract.",
```

Comment:
“Tool for retrieving events data (cluster snapshots and owner nonce) from the SSV network contract.”